miio/yeelight.py

```diff
@@ -419,7 +562,66 @@ def dump_ble_debug(self, table):
         """
         return self.send("ble_dbg_tbl_dump", {"table": table})
 
-    def set_scene(self, scene, *vals):
+    @command(
+        click.argument("name", type=str),
+        click.option(
+            "--lamp", type=int, required=False, default=YeelightSubLightType.Main
+        ),
+        default_output=format_output("Start {name} color flowing"),
+    )
+    def start_color_flowing(self, name, lamp=YeelightSubLightType.Main):
+        """Start color flowing."""
+        flow = get_flow_info(name)
+        if flow is None:
+            raise YeelightException("Unknown flow")
+        if lamp == YeelightSubLightType.Background:
+            if not self.model_specs.has_background_light:
+                raise YeelightException("Background lamp not supported")
+        elif flow.is_for_color and not self.model_specs.supports_color:
+            raise YeelightException("Lamp not supported this color flow")
+        return self.send(
+            SUBLIGHT_PROP_PREFIX[lamp] + "start_cf",
+            [flow.count, flow.action, flow.expression_string],
+        )
+
+    @command(
+        click.option(
+            "--lamp", type=int, required=False, default=YeelightSubLightType.Main
+        ),
+        default_output=format_output("Stop current color flowing"),
+    )
+    def stop_color_flowing(self, lamp=YeelightSubLightType.Main):
+        """Stop color flowing."""
+        if (
+            lamp == YeelightSubLightType.Background
+            and not self.model_specs.has_background_light
+        ):
+            raise YeelightException("Background lamp not supported")
+        return self.send(SUBLIGHT_PROP_PREFIX[lamp] + "stop_cf")
```

Comment:
Are you sure both the main and the background light have the same features? Colors, color flows etc.? I had the idea that the background one was just a color temperature controllable, or maybe dimmable, light?